druid/src/widget/slider.rs

```diff
@@ -56,6 +58,24 @@ impl Slider {
         self.max = max;
         self
     }
+
+    /// Builder-style method to set the stepping.
+    ///
+    /// The default step size is `0.0` (smooth).
+    pub fn with_step(mut self, step: f64) -> Self {
```

Comment:
can we also get a setter method

Reply:
Do we need one? We don't have a setter for the range either. I'd like to keep the API consistent here I think.

Reply:
normally in druid we have a setter method for every builder style. I am not sure why Slider was skipped